.github/workflows/release.yml

```diff
@@ -0,0 +1,67 @@
+---
+name: Release
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+  push:
+    branches:
+    - main
+
+env:
+  DESTDIR: /tmp/build
+  LUA_LIB_DIR: /usr/local/openresty/lualib
+  CARGO_NET_GIT_FETCH_WITH_CLI: true
+
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    outputs:
+      published: ${{ steps.release.outputs.published }}
+      release-git-tag: ${{ steps.release.outputs.release-git-tag }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Release
+      id: release
+      uses: ahmadnassri/action-semantic-release@v2.1.10
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  artifacts:
+    needs: release
+    name: Create Release Artifacts
+    runs-on: ubuntu-latest
+    container: rustembedded/cross:${{ matrix.target }}
+    strategy:
+      matrix:
+        target:
+          - aarch64-unknown-linux-musl
+          - x86_64-unknown-linux-gnu
+          - x86_64-unknown-linux-musl
+          - aarch64-unknown-linux-gnu
```

Comment:
Do we need `x86_64-unknown-linux-gnu:centos` in the matrix for Glibc 2.17?

Reply:
That would work for x86 but wasn't available for aarch64 so instead https://github.com/Kong/atc-router/pull/16/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R35 covers both options

Reply:
Actually, we might indeed want that. I see both `aarch64-unknown-linux-gnu:centos` and `x86_64-unknown-linux-gnu:centos` available on https://github.com/cross-rs/cross/blob/main/README.md#supported-targets (I must have missed it when I originally looked).

Reply:
That's why I was asking.

Reply:
Ran into some challenges handling this with `cross`

```
    strategy:
      matrix:
        target:
          - x86_64-unknown-linux-gnu:centos
#          - aarch64-unknown-linux-gnu:centos
#          - x86_64-unknown-linux-musl
#          - aarch64-unknown-linux-musl

    steps:
      - uses: actions/checkout@v3
      - uses: actions-rs/toolchain@v1
        with:
          toolchain: stable
          target: ${{ matrix.target }}
          override: true
```

Returns
```
nfo: using existing install for 'stable-x86_64-unknown-linux-gnu'
info: override toolchain for '/home/runner/work/atc-router/atc-router' set to 'stable-x86_64-unknown-linux-gnu'

  stable-x86_64-unknown-linux-gnu unchanged - rustc 1.65.0 (897e37553 2022-11-02)

/home/runner/.cargo/bin/rustup target add --toolchain stable x86_64-unknown-linux-gnu:centos
error: toolchain 'stable-x86_64-unknown-linux-gnu' does not contain component 'rust-std' for target 'x86_64-unknown-linux-gnu:centos'
note: not all platforms have the standard library pre-compiled: https://doc.rust-lang.org/nightly/rustc/platform-support.html
Error: The process '/home/runner/.cargo/bin/rustup' failed with exit code 1
```

Reply:
https://github.com/cross-rs/cross/blob/main/README.md#supported-targets has me looking at creating a `Cross.toml`. If this is the wrong approach please redirect me :)